src/devices/MAX31856/samples/README.md

```diff
@@ -0,0 +1,4 @@
+# TODO: This needs to be determined
```

Comment:
TODO: remove this file :-) It's not needed as you've placed everything into the main read me